make.go

```diff
@@ -236,12 +236,16 @@ func createGitRepository(debsrc, gopkg, orig string) (string, error) {
 		return dir, err
 	}
 
-	if err := runGitCommandIn(dir, "config", "user.name", getDebianName()); err != nil {
-		return dir, err
+	if debianName := getDebianName(); debianName == "TODO" {
```

Comment:
user.Current() returns an os.User object with user.Name set to "" if, in fact, the user has no full name set. I didn't give this account a full name because it's a throw-away VM guest.

I'll follow up with a patch to return this as "TODO". A similar error exists in getDebianEmail; I'll fix that one too.